### DISCUSSION TITLE
Shifting interpolation

### DISCUSSION THREAD

**Discussion by IvonaTau** [JCBrouwer/maua-stylegan2#6] [General]
Hi! 

I found this example through the link in README and I was wondering how did you generate those "shifting" latents that look like they were moving from left to right? 
Looks really cool and I could use something similar in my project :) 

https://www.youtube.com/watch?v=nRQavNZqd_k&list=PLkain1QGMwiWndQwr3U4shvNpoFC21E3a&index=2

**Comment by JCBrouwer** [comment]
You can find the code for this video (& 2 more) in the `audioreactive/examples` folder.

The scrolling effect is implemented in [`tauceti.get_bends()`](https://github.com/JCBrouwer/maua-stylegan2/blob/master/audioreactive/examples/tauceti.py#L94)
